src/main/java/net/mcreator/element/converter/fv26/CommandArgumentBlockFixer.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2021, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.element.converter.fv26;
+
+import com.google.gson.JsonElement;
+import net.mcreator.element.GeneratableElement;
+import net.mcreator.element.converter.IConverter;
+import net.mcreator.element.types.Command;
+import net.mcreator.workspace.Workspace;
+
+public class CommandArgumentBlockFixer implements IConverter {
+
+	@Override
+	public GeneratableElement convert(Workspace workspace, GeneratableElement input, JsonElement jsonElementInput) {
+		Command command = (Command) input;
+		command.argsxml = "<xml><block type=\"args_start\" deletable=\"false\" x=\"40\" y=\"40\"></block></xml>";

```

Comment:
You should also by default convert existing commands to use procedure call block to call the procedure they used before

Reply:
I am not against the idea. However, every command using the old command parameter getter block (so the old dependency too) will not have a working command command in the game, but they will not be able to notice until they test it as it will not produce errors inside MCreator. The best thing would be to convert everything, procedures and add blocks inside the new Blockly editor, but it may not be possible with the current way the converter works. We should at least warns the user its commands will not work anymore.

Reply:
So the new system does not support calling procedure on command as before?

Reply:
What I meant is this new system does not support the old command dependency where you need to write the number of the argument to get it as a string. Bow, you need the name of the argument you wrote inside the command argument block you placed inside the Blockly panel. This system still uses a procedure to execute actions.

Reply:
I see. I would say make a converter that converts procedures that do not use command arguments. For complex procedures, we will add release note to redesign those commands.

It would also be worth considering adding command argument "anything" to support sort of functionality there is right now, possibly

Reply:
> It would also be worth considering adding command argument "anything" to support sort of functionality there is right now, possibly

I will add it in the other PR after you merge this first PR into a temporary branch.

Reply:
It may be good to have it in this one, so the converter is valid, or at least make converter convert all but anything parameter here